lib/datasource/github.js

```diff
@@ -47,16 +44,14 @@ async function getDependency(purl) {
   if (!versions) {
     return null;
   }
-  // Filter by semver if no versionScheme provided
-  if (options.clean === 'true') {
-    versions = versions.map(version => version.replace(/^v(\d)/, '$1'));
-  }
   const dependency = {
     repositoryUrl: 'https://github.com/' + repo,
-    versions: {},
   };
-  versions.forEach(version => {
-    dependency.versions[version] = {};
-  });
+  // TODO: should Github receive the versionScheme as input?
+  // versions should be already sorted by GIT
+  dependency.releases = versions.map(version => ({
```

Comment:
I assume it's intentional to make releases now an array containing `version` as a field rather than an object with `version` as the key? Maybe we can separate that into its own commit/PR first and reduce the number of files that this PR changes.